index.js

```diff
@@ -77,6 +77,7 @@ function replaceRequireEnsure(chunkId) {
 							document.dispatchEvent(new CustomEvent(
 								'chunk-progress-webpack-plugin', {
 									detail: {
+										originalEvent: event,
 										loaded: loaded,
```

Comment:
Sorry busy week didn't have time to review earlier. This event is not the original from requesting for header with file size its the event from subsequential calls for progress.
The event from https://github.com/seleb/chunk-progress-webpack-plugin/blob/23bf0246a819894394841c1d17dd544a9b425721/index.js#L65 is a lot more usefull for reporting file size.